src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelCsdlSchemaXmlWriter.cs

```diff
@@ -157,6 +157,14 @@ internal override void WritePropertyRefElement(IEdmStructuralProperty property)
             this.WriteEndElement();
         }
 
+        internal override void WritePropertyRefElement(string keyname, string keyAlias)

```

Comment:
```suggestion
        internal override void WritePropertyRefElement(string keyName, string keyAlias)

```
nit: on naming consistency